src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -589,7 +589,11 @@
 
 		<!-- domain.c: mono_defaults.thread_class -->
 		<!-- FIXME: -->
-		<type fullname="System.Threading.Thread" preserve="fields" />
+		<type fullname="System.Threading.Thread" preserve="fields">
+			<!-- threads.c -->
+			<method name="CreateAutoreleasePool" />
+			<method name="DrainAutoreleasePool" />
```

Comment:
Hmm, this should be preferably in apple specific additional descriptor xml file

Reply:
Thanks @marek-safar. I was hoping that was the solution.